### DISCUSSION TITLE
Expenses plugin - export expense type?

### DISCUSSION THREAD

**Discussion by vic-t** [kimai/kimai#2353] [General]
I'm currently reviewing the expenses plugin as I really like the idea of including even more billable information in one place. 

Unfortunately, however, the exporting possibilities don't seem to match the capabilities promised by the user interface. While it's possible to define different kinds of categories of expenses, such as Travel, Catering etc., it seems these categories are not part of the data export and thus also not part of the invoice. As a matter of fact, from the exported data alone it does not seem possible to differentiate timesheet entries and expenses. In my opinion, this reduces the benefit of the plugin considerably. 

Am I missing something?

**Comment by kevinpapst** [comment]
Category and Type (timesheet, expense) are exported IMO. But the exported data depends on the invoice/export template.

**Reply by vic-t** [reply]
I just tried this on your Plugin Demo, the exported CSV looks like this (for one timesheet entry and one travel expense): 
```
"Date","From","To","Duration","Rate","Internal rate","User","Name","Customer","Project","Activity","Description","Exported","Tags","Hourly rate","Fixed rate"
"2021-02-14","13:01","15:01","7200"," EUR 300.00 "," EUR 300.00 ","admin","admin","MyCustomer","MyProject","MyActivity","test","Open",""," EUR 150.00 ",""
"2021-02-13","00:00","00:00","0"," EUR 45.00 "," EUR 45.00 ","admin","admin","MyCustomer","MyProject","MyActivity","Travelwerwerwerewr","Open","",""," EUR 0.30 "
```

It also doesn't seem to be part of the Excel or PDF template, for that matter. 

Do you have an export template that would export it?

**Reply by kevinpapst** [reply]
I used them in many custom PDF export templates which I have built for customers, but seems you are right and the global Spreadsheet exporter (CSV, XLSX) does not include these columns.

**Reply by vic-t** [reply]
So we concluded that these columns are currently not present in the export which diminishes the use of the paid Expenses plugin. Can you confirm that you will improve on that?

**Reply by kevinpapst** [reply]
Adding more default columns to the spreadsheet exporter is no problem. Please open a feature request.

**Reply by vic-t** [reply]
By the way, thanks for the exchange of thoughts and ideas today, and your help, of course.
